migrations/1616528363530-add_cashbox_batches.js

```diff
@@ -0,0 +1,26 @@
+var db = require('./db')
+
+exports.up = function (next) {
+  var sqls = [
+    `create table cashbox_batches (
+      id uuid PRIMARY KEY,
+      device_id text REFERENCES devices (device_id),
+      created timestamptz NOT NULL default now()
+    )`,
+
+    `ALTER TABLE bills ADD COLUMN legacy boolean DEFAULT false`,
+
+    `ALTER TABLE bills ADD COLUMN cashbox_batch_id int`,
```

Comment:
`Int` here should be UUID

Reply:
I think that was fixed here https://github.com/lamassu/lamassu-server/pull/679/files